src/routes/custom-labels-route/custom-labels-route.js

```diff
@@ -29,11 +29,7 @@ const SettingsCustomLabelsRoute = ({
 
   useEffect(() => {
     getCustomLabels(match.params.kbId);
-  }, []);
-
-  useEffect(() => {
-    getCustomLabels(match.params.kbId);
-  }, [match.params.kbId]);
+  }, [getCustomLabels, match.params.kbId]);
```

Comment:
Please make sure there aren't duplicate requests for custom labels

Reply:
I see that there are no duplicate requests.